package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Project Information Component for Igniter",
   "main": "index.js",
   "scripts": {
-    "test": "jest --coverage",
+    "test": "jest --coverage --detectOpenHandles",
```

Comment:
For circleci the coverage flag is no needed. This related with the optimization of the test process. the test with the coverage flag could be in another scrip so can be run when needed .

Reply:
@despinoz made the changes you suggested.